release/nightly_tests/shuffle/shuffle_compute_autoscaling.yaml

```diff
@@ -0,0 +1,21 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+aws:
+    BlockDeviceMappings:
+        - DeviceName: /dev/sda1
+          Ebs:
+            VolumeSize: 500
+
+head_node_type:
+    name: head_node
+    instance_type: i3.4xlarge
+    resources: {"object_store_memory": 21474836480}
```

Comment:
I guess this is copied from other config, but we can remove this now since SIGBUS is properly handled